### DISCUSSION TITLE
npm does not support Node.js v14.8.0

### DISCUSSION THREAD

**Discussion by Shawn1174q** [rdudhagra/Sand-Table#28] [Troubleshooting] [answered]
I am getting that npm does not support Node.js.v14.8.0. I read in other posts that you encourage 14.8 as it is a proven stable release. When I go to my pi's ip in a windows browser, a  nginx greeting comes up proving that nginx is installed correctly. I believe if I can get npm to the right version that syncs with Node.js.v14.8.0, everything will work properly. 
EDIT: I updated to v16.3.0 and seems to solve the problem except now, I am getting the 4th picture as a result.
![sand1](https://user-images.githubusercontent.com/13476885/150265082-71f20c68-f051-4cdf-a46b-4d070d1f9617.JPG)
![sand2](https://user-images.githubusercontent.com/13476885/150265086-00ed8c9f-3f74-487e-ac26-5f55031c76ac.JPG)
![sand3](https://user-images.githubusercontent.com/13476885/150265089-8b5f1645-b624-4084-9b16-c602eccdcfd8.JPG)
![sand4](https://user-images.githubusercontent.com/13476885/150452254-99f6bfa8-31c8-4b52-a8a9-33dc050818c6.JPG)

**Comment by rdudhagra** [comment, marked as answer]
Hey Shawn, apologies for the late reply...here are a few things to try:

1. With `Node 14.8.0`: What version of `npm` do you have installed? That might be the source of the problem. According to [this website](https://nodejs.org/en/download/releases/) you should have `npm 6.14.7` (I'd verify with my table but unfortunately I'm away from home for the foreseeable future). 
2. With `Node 16.3.0`: Did you run `npm install`? If I remember correctly this is going to cause an error because the node version messes with some of the required packages, but if it works for you then problem solved!

Let me know if this solves the problem!

**Reply by rdudhagra** [reply]
If you do end up needing to downgrade `npm`, [this Stack Overflow post](https://stackoverflow.com/questions/46415343/downgrade-npm-to-an-older-version) should do the trick. Let me know if this solved your issue and I'll update the build guide.
